### ISSUE TITLE
Dark Beast in Mourner Tunnels

### ISSUE THREAD

**Issue by Rallik-Pog** [PTYB/SlayerIssueTracker#198]
**Issue:** Dark Beast in Mourner Tunnels unsupported.
**Current behavior:** Unsupported
**Expected behavior:** Use Slayer ring, or slayer ring eternal to teleport to dark beasts. (Will also need a teleport out) Or give the option to skip the task.
**Screenshots/Video:** 
![image](https://github.com/PTYB/SlayerIssueTracker/assets/118016227/c37b529f-0ab3-4c52-af7e-5f93b8863943)

**Master:** Konar
**Task:** See below
**Additional information:** you will need a teleport in and out

![image](https://github.com/PTYB/SlayerIssueTracker/assets/118016227/01c1dc6c-71d6-4764-9702-264a11df989b)

west tile: 
![image](https://github.com/PTYB/SlayerIssueTracker/assets/118016227/cc521f13-abfd-41cd-b084-64257d29c074)
north tile: 
![image](https://github.com/PTYB/SlayerIssueTracker/assets/118016227/4d69aa3f-c3ff-4d03-a930-6456a702a4d1)

**Comment by Rallik-Pog** [comment]
added tiles

**Comment by PTYB** [comment]
Skip task option should already be released.

**Comment by Rallik-Pog** [comment]
Yes it is, but Dark Beasts in the Mourner Tunnels doesn't have its own place on the monster menu to be able to choose to skip task.
